Lib/ftplib.py

```diff
@@ -75,13 +75,14 @@ class FTP:
     '''An FTP client class.
 
     To create a connection, call the class using these arguments:
-            host, user, passwd, acct, timeout
+            host, user, passwd, acct, timeout, source_address, encoding
 
     The first four arguments are all strings, and have default value ''.
-    timeout must be numeric and defaults to None if not passed,
-    meaning that no timeout will be set on any ftp socket(s)
+    The fifth parameter ´timeout´ must be numeric and defaults to None if not
```

Comment:
nitpick: I would prefer tot not "promote" the usage of postional parameters by not giving their position.

```suggestion
    The parameter ´timeout´ must be numeric and defaults to None if not
```